recipes/mbuild/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "mbuild" %}
+{% set version = "0.10.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mosdef-hub/mbuild/archive/{{ version }}.tar.gz
+  sha256: eb9878f550591a68d5404f5b173ad6343a236fbe9eca3331e53e8d8fc918e658
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - protobuf
+    - python >=3.6
+  run:
+    - mdtraj
+    - nglview >=2.7
+    - {{ pin_compatible('numpy') }}
```

Comment:
You don't need pin compatible here. Just numpy.

Reply:
```suggestion
    - numpy
```